common/src/main/java/com/wynntils/core/features/overlays/OverlayPosition.java

```diff
@@ -30,6 +35,24 @@ public OverlayPosition(
         this.horizontalOffset = horizontalOffset;
     }
 
+    public OverlayPosition(String string) {
```

Comment:
Maybe better make this a static factory  method `from` than a constructor?

Where is this used? In json config file parsing? I'm a bit worried about the ordering requirement for the fields, it will look like you can give them in any order but this will not be accepted.

Reply:
It is used in config parsing reflection, for the WynntilsConfigCommand. I didn't make it accept any order, as we will only use this for the command, and the actual config gui will not need this.

Reply:
(See the ConfigHolder for usage.)

Reply:
I see. Then I guess it is okay. We can always extend it later, if needed. But you can still make it a factory method instead of a constructor?

Reply:
That would require changing the ConfigHolder value parsing to include even more reflection stuff? Isn't it fine to leave it as a string constructor?

Reply:
Fine, leave it as a constructor.